src/cpu.h

```diff
@@ -7,15 +7,15 @@
 #include <stdio.h>
 #include <time.h>
 
-#define MEMORY_SIZE 4096
-#define REGISTER_COUNT 16
-#define STACK_LEVELS 16
-#define KEY_SIZE 16
-#define VIDEO_HEIGHT 32
-#define VIDEO_WIDTH 64
+#define MEMORY_SIZE		  4096
+#define REGISTER_COUNT	  16
+#define STACK_LEVELS	  16
+#define KEY_SIZE		  16
+#define VIDEO_HEIGHT	  32
+#define VIDEO_WIDTH		  64
```

Comment:
This does not seem right, indentation should be the same for all defines here

Reply:
That's Github messing something up, see my quote above:

> I would suggest not looking at the changes, since github does not display everything correctly. Instead looking at the source directly should be better.

Reply:
Nvm this then, did overlook that sentence